### PR TITLE
Remove Mqueue Adapter

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Mqueue Adapter - it's unused code in component HMIMessageHandler, so I deleted it.

### CODE REVIEW

**Comment by LitvinenkoIra** [comment]
@LuxoftAKutsan @dtrunov @VVeremjova @anosach-luxoft @AGaliuzov please review

File: src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests [hunk 1]
```diff
@@ -15,63 +15,63 @@
                         'configure.in'
                       ],
                       {
-                        'AM_MAKEFILE_INCLUDE' => 1,
+                        'AM_PROG_FC_C_O' => 1,
```
Comment (LuxoftAKutsan) [inline src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests]: @LitvinenkoIra I suppose this files are autogenerated and should not be committed

**Comment by LuxoftAKutsan** [comment]
@LitvinenkoIra note that build server failed build on you commit.
Failed unit tests : HMIMessageHandlerImplTest.SendMessageToHMI_Success 
Is was failed before you changes?

**Comment by VVeremjova** [comment]
@LitvinenkoIra Check HMIMessageHandlerImplTest

**Comment by LitvinenkoIra** [comment]
@LuxoftAKutsan 
Failed unit tests : HMIMessageHandlerImplTest.SendMessageToHMI_Success
Yes, it was failed before my changes

**Pull request closed without merging**